apps/dialtone-documentation/docs/components/toggle.md

```diff
@@ -46,24 +48,13 @@ The Toggle component acts as a way to allow the User to switch between two mutua
 
 <code-well-header>
   <fieldset class="d-stack8">
```

Comment:
can we use stack component with `as="fieldset"` instead of all these `d-mr6`?

Reply:
Changed the `fieldset` to `dt-stack` but for the toggles with labels I still need to use some class to set the margin, I can't use stack for that because button and label era wrapped in a `div`.

Reply:
k no prob, thanks!